.travis.yml

```diff
@@ -0,0 +1,46 @@
+sudo: false
+
+language: php
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+
+git:
+  depth: 5
+
+php:
+  - 5.3
+  - 5.4
+  - 5.5
+  - 5.6
+  - 7.0
+  - hhvm
+
+env:
+  - SYMFONY_VERSION="2.3.*"
+  - SYMFONY_VERSION="2.7.*"
+  - SYMFONY_VERSION="2.8.*"
+  - SYMFONY_VERSION="3.0.*"
+
+matrix:
+  allow_failures:
+    - env: SYMFONY_VERSION="2.3.*"
+    - env: SYMFONY_VERSION="3.0.*"
+  exclude:
+    # Symfony 3.0 requires PHP >= 5.5.9
+    - php: 5.3
+      env: SYMFONY_VERSION="3.0.*"
+    - php: 5.4
+      env: SYMFONY_VERSION="3.0.*"    
+
+addons:
+  apt:
+    packages:
+      - php5-sqlite
```

Comment:
The tests are broken with Symfony 2.3 and 3.0 at this moment, so I kept the `allow_failures` part but it should be removed in the future.